decide/booth/test-selenium.py

```diff
@@ -171,3 +171,110 @@ def test_voting_list(self):
         self.assertTrue(
             len(self.driver.find_elements(By.CLASS_NAME, "voting-closed")) == 1
         )
+
+
+class OpinionsViewTestCase(StaticLiveServerTestCase):
+    def setUp(self):
+        # Crea un usuario admin y otro no admin
+        self.base = BaseTestCase()
+        self.base.setUp()
+
+        user = User.objects.get(username="noadmin")
+        # Crea la votacion
+        question = Question.objects.create(desc="Test question")
+        voting1 = Voting.objects.create(
+            name="Test1",
+            desc="Test",
+            start_date=timezone.make_aware(datetime.datetime(2023, 10, 11)),
+            end_date=timezone.make_aware(datetime.datetime(2023, 10, 12)),
+            question_id=question.id,
+        )
+        auth = Auth.objects.get_or_create(
+            url=settings.BASEURL, defaults={"me": True, "name": "test auth"}
+        )[0]
+        voting1.auths.add(auth)
+        Census.objects.create(voter_id=user.id, voting_id=voting1.id)
+
+        app = SocialApp.objects.create(
+            provider="google",
+            name="Google",
+            client_id="test",
+            secret="test",
+        )
+        # Add the current site to the SocialApp's sites
+        app.sites.add(Site.objects.get_current())
+
+        # Opciones de Chrome
+        options = webdriver.ChromeOptions()
+        options.headless = False
```

Comment:
@Marchabar This needs to be set to True, otherwise the tests won't pass on the github actions

Reply:
Done!